vocabularies/Org.OData.Core.V1.xml

```diff
@@ -509,15 +509,23 @@ Any simple identifier | Any type listed in `Validation.OpenPropertyTypeConstrain
         <Annotation Term="Core.LongDescription" String="The annotation value will usually be an expression, e.g. using properties of the binding parameter type for instance-dependent availability, or using properties of a singleton for global availability. The static value `null` means that availability cannot be determined upfront and is instead expressed as an operation advertisement." />
       </Term>
 
+      <Term Name="RequiresExplicitBinding" Type="Core.Tag" DefaultValue="true" AppliesTo="Action Function">
+        <Annotation Term="Core.Description" String="This bound action or function is only available on model elements annotated with the ExplicitOperationBindings term." />
+      </Term>
+
+      <Term Name="ExplicitOperationBindings" Type="Collection(Edm.String)">

```

Comment:
I'd introduce a type definition similar to the existing `QualifiedActionName`. 

Also: would this be just the name, allowing all function overloads with a matching binding parameter type, or would you also want to identify specific function overloads?